JobConfig/digitize/prolog.fcl

```diff
@@ -156,15 +180,13 @@ Digitize: {
   #
   # trigger selections that can be used to finalize the triggerOutput selection, depending on digitization mode and source
   #
-  SignalTriggers : [ "tprDe_highP_stopTarg*", "cprDe_highP_stopTarg*" ]  # events with 'high' momentum tracks with good KinKal fits
+  SignalTriggers : [ "tpr*_highP*", "cpr*_highP*", "apr_highP*" ]  # events with 'high' momentum tracks with KinKal fits
```

Comment:
Should this include the MPR triggers as well? If not, should this also remove the TPR triggers as we've moved to APR as the default?

Reply:
Good point. I'll remove TPR.